### PR TITLE
Add whitespace before `else`

### PR DESCRIPTION
**Pull request by stevapple**

### CODE REVIEW

File: Sources/SwiftSyntax/generated/SyntaxCollections.swift
Comment (kimdv) [inline Sources/SwiftSyntax/generated/SyntaxCollections.swift:1]: Is those changes part of fixing the leading whitespace?

Comment (stevapple) [reply Sources/SwiftSyntax/generated/SyntaxCollections.swift:1]: I guess not. They may come from missing regeneration from a prior PR, or maybe caused by rebasing PRs on top of `main`.

Comment (kimdv) [reply Sources/SwiftSyntax/generated/SyntaxCollections.swift:1]: It was part of this PR: https://github.com/apple/swift-syntax/pull/1482 
More specific this commit: 0e21bb78329bfdb07fdc223e6fb79fd2582dddc3

But I see that some of the code is correctly generated in that PR 🫣

Comment (stevapple) [reply Sources/SwiftSyntax/generated/SyntaxCollections.swift:1]: Maybe caused by missing regeneration after rebase?

Comment (ahoppen) [reply Sources/SwiftSyntax/generated/SyntaxCollections.swift:1]: Yes, I think #1455 and #1482 raced here and the code for the new nodes wasn’t properly generated. My bad.

**Review by ahoppen — APPROVED** [review]
Thanks. Looks good to me.

**Comment by ahoppen** [comment]
@swift-ci Please test

**Pull request merged**